views/index.handlebars

```diff
@@ -8,7 +8,7 @@
             thoughts</h5>
         </div>
         <div class="row center">
-          <a href="/signup" id="startButton" class="btn-large waves-effect waves-light grey darken-4">Start Journaling</a>
```

Comment:
you are deleting the id name for the onclick function you wrote above. this will break the onclick.